lib/renderer/override.js

```diff
@@ -161,11 +161,14 @@ window.open = function(url, frameName, features) {
 // Use the dialog API to implement alert().
 window.alert = function(message, title) {
```

Comment:
The problem with your code is if you passed `undefined` it still would be replaced by the default. So in your example passing `undefined` as the message would give me a blank `alert()` dialog.  In a browser `alert()` will give me a blank dialog, but `alert(undefined)` will give me a dialog that says `undefined`